src/scene/text/TextStyle.ts

```diff
@@ -535,6 +560,13 @@ export class TextStyle extends EventEmitter<{
             }
         });
     }
+
+    private _isFillStyle(value: FillInput): value is FillStyle
+    {
+        // eslint-disable-next-line no-eq-null, eqeqeq
+        return (value != null
```

Comment:
Why violate the lint here?

Reply:
I think being verbose here is okay. Style should only be over ridden as a necessity. You could try using the coalesce operator.

```ts
(value ?? null) !== null
```